.github/pull_request_template.md

```diff
@@ -1,62 +1,61 @@
-<!--
-### How to use
-Leave the headings unless they don't apply to your PR, replace commented out text (surrounded with <!–– and ––>) with text describing your PR.
-NOTE: Please grant permission for repository maintainers to edit your PR.
-It is EXTREMELY common for PRs to be held up due to trivial changes being requested and the author being unavailable to make them.
+<!-- HOW TO USE: Under each "#### Heading" below, enter information relevant to your pull request.
+Leave the headings unless they don't apply to your PR, and remove the comment blocks (surrounded with <!–– and ––>) when you are done.
+
+NOTE: Please grant permission for repository maintainers to edit your PR.  It is EXTREMELY common for PRs to be held up due to trivial changes being requested and the author being unavailable to make them.
 -->
 
 #### Summary
-<!--
-A one-line description of your change that will be extracted and added to the [project changelog](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt).
 
-The format is (ignore the square brackets): ```SUMMARY: [Category] "[description]"```
+<!-- This section should consist of exactly one line, formatted like this:
+
+SUMMARY: [Category] "[Briefly describe the change in these quotation marks]"
+
+Do not enter the square brackets [].  Category must be one of these:
+
+- Features
+- Content
+- Interface
+- Mods
+- Balance
+- Bugfixes
+- Performance
+- Infrastructure
+- Build
+- I18N
 
-The categories to choose from are:
+For more on the meaning of each category, see:
+https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CHANGELOG_GUIDELINES.md
 
-* Features
-* Content
-* Interface
-* Mods
-* Balance
-* Bugfixes
-* Performance
-* Infrastructure
-* Build
-* I18N
+If approved and merged, your summary will be added to the project changelog:
+https://github.com/CleverRaven/Cataclysm-DDA/blob/master/data/changelog.txt
 
-Example: ```SUMMARY: Content "Adds new mutation category 'Mouse'"```
+Example SUMMARY lines: (only enter ONE!)
 
-See the [Changelog Guidelines](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/CHANGELOG_GUIDELINES.md) for explanations of the categories.
+SUMMARY: Features "Implement item pockets"
+SUMMARY: Content "Add nested maps to 4 basements"
+SUMMARY: Bugfixes "Fix bugs and increase performance of 3D shadowcasting"
+SUMMARY: Infrastructure "Improve test suite for item info panel"
```

Comment:
The reason I didn't have valid inline examples like this is that they will confuse the validator, these examples existing inside a comment will either satisfy it or confuse it, and they will definitely not do the right thing when they arrive at the summary extractor.

Reply:
Thank you - I had not considered the effec this would have on validation. These examples are removed in the latest version.